webui/react/src/components/DatePickerFilter.tsx

```diff
@@ -1,31 +1,23 @@
 import { DatePicker } from 'antd';
+import { PickerProps } from 'antd/es/date-picker/generatePicker';
 import { Dayjs } from 'dayjs';
 import React from 'react';
 
 import Label from './Label';
 import css from './SelectFilter.module.scss';
 
 interface Props {
-  className?: string;
   label: string;
-  onChange?: (date: Dayjs | null) => void;
-  value?: Dayjs;
 }
 
-const DatePickerFilter: React.FC<Props> = ({ className = '', label, onChange, value }: Props) => {
-  const classes = [ className, css.base ];
-
-  return (
-    <div className={classes.join(' ')}>
-      <Label>{label}</Label>
-      <DatePicker
-        showNow={false}
-        showTime
-        value={value}
-        onChange={onChange}
-      />
-    </div>
-  );
-};
+const DatePickerFilter: React.FC<PickerProps<Dayjs> & Props> =
```

Comment:
**nit:** if you define the `Props` interface to be the following it might keep the component function a little bit simpler:
```
interface Props & PickerProps<Dayjs> { ... }

const DatePickerFilter: React.FC<Props> = ({ label, ...props }: Props) => { ... };
```